content/en/docs/contributing/development.md

```diff
@@ -26,6 +26,47 @@ cloud IDE:
 
 Gitpod automatically installs the repo-specific packages for you.
 
+## Cloning the Repository
+
+For first-time contributors, it is recommended to clone the repository using SSH instead of HTTPS to avoid common errors related to authentication.
+
+### Cloning with SSH
+
+To clone the repository using SSH, follow these steps:
+
+1. **Generate an SSH Key (if you don't have one):**
+   - Open a terminal and run:
+     ```bash
+     ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+     ```
+   - Follow the prompts to save the key to the default location.
+
+2. **Add your SSH key to the SSH agent:**
+   - Start the SSH agent:
+     ```bash
+     eval "$(ssh-agent -s)"
+     ```
+   - Add your SSH key:
+     ```bash
+     ssh-add ~/.ssh/id_rsa
+     ```
```

Comment:
To make this work reliably they need to add the ssh agent part also to their bashrc or zshrc. If I remember correctly using a ssh agent is not mandatory, so for new comers it might be easier to be prompted to type the SSH key passphrase every time. maybe mark it as optional